### ISSUE TITLE
tag that uses slash as part of the style will not be replaced to <div/>

### ISSUE THREAD

**Issue by sohoffice** [sohoffice/angular-universalize-email#2]
For example, the mat-card-title will have styles similar to the below

```
<mat-card-title style="font:400 56px/56px Roboto, &apos;Helvetica Neue&apos;, sans-serif;">
  ...
</mat-card-title>
```

The tag name `mat-card-title` should be replaced with `div`.

However, the slash in the font name is causing issue to the parser, so the open tag <mat-card-title> remain unchanged. The closing tag </mat-card-title> is successfully replaced, which creates unclosed tag in the end.

**Comment by sohoffice** [comment]
Fixed in v1.1.1

**Comment by sohoffice** [comment]
Fixed in v1.1.1.
